index.ts

```diff
@@ -0,0 +1,28 @@
+/* Part 1 */
+
+function isChanged(n: number): boolean{
+    if (n % 3 == 0)
+        return true;
+    if (n % 5 == 0)
+        return true;
+    return false;
+}
+
+function getMessage(n: number): string{
```

Comment:
I quite like this function. I think it could be improved a bit more and instead this could also return the number (to string eventually) instead of that error message. This way, there is no need for is changed.